loss/loss.py

```diff
@@ -39,4 +43,13 @@ def recall(pred,target):
     """
     TP = np.sum(np.logical_and(pred == 1, target == 1))
     FN = np.sum(np.logical_and(pred == 0, target == 1))
-    return TP/(TP + FN)
\ No newline at end of file
+    return TP/(TP + FN)
+
+
+class CrossEntropyLoss2d(nn.Module):

```

Comment:
What is the problem of this function?

Reply:
it does not work well because it cannot match different dimensions properly, I will work on it

Reply:
> What is the problem of this function?


it does not work well because it cannot match different dimensions properly, I will work on it